docker-compose.yml

```diff
@@ -11,21 +11,39 @@ services:
       - '27017:27017'
     expose:
       - '27017'
+    networks:
+      - scout-net
 
   scout-cli:
     image: clinicalgenomics/scout
     container_name: scout-cli
-    links:
-      - mongodb
     command: scout --host mongodb setup demo
+    volumes:
+      - ./scout:/home/worker/app/scout
+      - ./scripts:/home/worker/app/scripts
```

Comment:
Personally I have mounted the mongo data as a volume as well to keep the data persistent when I take down the entire app. Do you think I should include this as well in the PR?

Reply:
Mmm, the original purpose of this file was to give a quick insight on how Scout works, by setting up a demo instance and providing a server so users could test without having to install/uninstall anything on their own. So I don't think this would be the right place to define persistent volumes, like the database storage folder. 

But perhaps we could document how to include such a volume in the documentation?

Reply:
Now I'm curious, do you plan to use such a setup in production or you already have something like this up and running there in Lund?

Reply:
Ah I see. Since the purpose of the compose file are demoing and not development I agree with you that mounting the database data to the host files system are unnecessary. I can add something about it in the README then.

We have talked about using a containerized version of Scout in production but we havnt started working on transitioning yet and are not clear on the exact method of deploying the container. What are your plans?

Reply:
We are also planning to move a lot of our software to containers. We had this idea since a while but now we are forced to move all our databases and apps (including Scout) to a new server farm since our current prod and stage server has end-of-life December this year. Since we won't have so much time for the moving I think we'll try to move as many softwares as we can but for the difficult ones we might need to install them again on conda environements, for a while at least. I don't know if we'll managed soon with Scout because it's connected to other databases and services (loqus, chanjo, matchmaker), but we'll try.

Our current plans include to deploy the images using Podman.